src/package/versionNumber.ts

```diff
@@ -31,7 +31,7 @@ export class VersionNumber {
    *
    * @param versionString a string in the format of major.minor like '3.2'
    */
-  public static parseMajorMinor(versionString: string): { major: number; minor: number } {
+  public static parseMajorMinor(versionString: string): { major: number | null; minor: number | null } {
```

Comment:
It's valid to have a `null:null` major minor version?

Reply:
Covering results from DB